src/common/services/api/donations.service.js

```diff
@@ -74,12 +74,12 @@ function DonationsService( cortexApiService, profileService, commonService ) {
         data:           data
       } )
       .map( ( response ) => {
-        angular.forEach( response['receipt-summaries'], ( item ) => {
-          let link = find( response.links, ( r ) => {
-            return r.uri.indexOf( item['transaction-number'] ) != -1;
-          } );
-          item['pdf-link'] = link;
-        } );
+        let links = filter(response.links, (link) => {
+          return link.rel == 'element';
+        }) || [];
+        for(let i=0; i < links.length; i++) {
+          response['receipt-summaries'][i]['pdf-link'] = links[i];
```

Comment:
Are we sure that the link elements are in the same order as the receipt-summaries? There's not a whole lot you can do on your end. Just want to make sure.